source/adapters/level_zero/command_buffer.cpp

```diff
@@ -958,7 +959,8 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferEnqueueExp(
   // Create a command-list to signal RetEvent on completion
   ur_command_list_ptr_t SignalCommandList{};
   UR_CALL(Queue->Context->getAvailableCommandList(Queue, SignalCommandList,
-                                                  false, false));
+                                                  false, NumEventsInWaitList,
+                                                  EventWaitList, false));
```

Comment:
There isn't a UR wait list for this command-list, it's ordering is enforced by calling ` Queue->executeCommandList(SignalCommandList, false, false);` after  the call on line 953
```
 ZE2UR_CALL(zeCommandQueueExecuteCommandLists,
             (ZeCommandQueue, 1, &CommandBuffer->ZeCommandList, ZeFence));
```

(see https://github.com/intel/llvm/blob/sycl/sycl/doc/design/CommandGraph.md#level-zero)